openfisca_core/entities.py

```diff
@@ -89,15 +89,14 @@ def init_variable_values(self, entity_object, entity_id):
                             value = holder.variable.possible_values[value]
                         except KeyError:
                             raise SituationParsingError(path_in_json,
-                                "'{}' is not a valid value for '{}'. Possible values are ['{}'].".format(
-                                    value, variable_name, "', '".join(holder.variable.possible_values.list)
-                                    ).encode('utf-8')
+                                u"'{}' is not a valid value for '{}'. Possible values are ['{}'].".format(
```

Comment:
Works great when Value has accents. Woudl it be possible to add a accentfull enum to country template to test if possible values works also ?